libvast/vast/detail/merge.hpp

```diff
@@ -13,25 +13,14 @@
 
 #pragma once
 
-#include "vast/value.hpp"
-#include "vast/concept/printable/vast/data.hpp"
+namespace vast::detail {
 
-namespace vast {
-
-struct value_printer : printer<value_printer> {
-  using attribute = value;
-
-  template <class Iterator>
-  bool print(Iterator& out, const value& v) const {
-    static auto const p = make_printer<data>{};
-    return p.print(out, v.data());
-  }
-};
-
-template <>
-struct printer_registry<value> {
-  using type = value_printer;
-};
-
-} // namespace vast
+/// Merges the contents of two containers into the first.
+template <class Container>
+void merge(Container& xs, Container&& ys) {
```

Comment:
Calling this `merge` seems slightly confusing given that there's already `std::merge` with different semantics. Maybe something like `combine()`, `append()` `append_into()`, `move_append()`?

Reply:
I like `append`. we use that for bitmaps, indexes, etc. already to convey non-commutative add.